src/freertos_drivers/net_cc32xx/CC32xxWiFi.hxx

```diff
@@ -219,6 +223,21 @@ public:
         return WlanState::OK;
     }
 
+    /** Used by unit tests to simulate wifi connection states. 
```

Comment:
Should we put a #if defined (GTEST) wrapper around this?

Reply:
done